buildenv/jenkins/JenkinsfileBase

```diff
@@ -368,12 +368,12 @@ def setup() {
 			setupEnv()
 
 			if (params.SDK_RESOURCE == 'nightly' && params.CUSTOMIZED_SDK_URL) {
-				CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"
-			} else if (params.CUSTOMIZED_SDK_URL) {
-				SDK_RESOURCE = "customized"
+			}else if (params.SDK_RESOURCE != 'customized'){
```

Comment:
`CUSTOMIZED_SDK_URL_OPTION = "-c '${params.CUSTOMIZED_SDK_URL}'"` is missing in the if block (params.SDK_RESOURCE == 'nightly' && params.CUSTOMIZED_SDK_URL)